src/components/AvatarWithIndicator.js

```diff
@@ -24,6 +24,9 @@ const propTypes = {
     /** To show a tooltip on hover */
     tooltipText: PropTypes.string,
 
+    /** All the user's policies */
+    policies: PropTypes.objectOf(PropTypes.object),
```

Comment:
How do I define this prop type? It's an object of policies keyed by policy id. The allPolicies prop type here doesn't make sense to me
https://github.com/Expensify/App/blob/18d373c5e47ea3b05dcea3ec9294fe0136d58aef/src/pages/home/sidebar/SidebarScreen/sidebarPropTypes.js#L7-L11

Reply:
I figured it out with fullPolicyPropTypes.